lib/dal/src/job/definition/dependent_values_update.rs

```diff
@@ -188,6 +194,68 @@ impl DependentValuesUpdate {
 
         let mut update_tasks = JoinSet::new();
 
+        // This is the core loop. Use both the individual and the management subscription to determine what to do next.
+        let needs_restart = tokio::select! {
+            result = self.listen(ctx, council, pub_council, dependency_graph, &mut status_updater, &mut update_tasks) => result,
+            management_result = self.listen_management(&mut management_council) => management_result,
+        }?;
```

Comment:
Core loop for dependent values update jobs. Either complete the job council gives us or see a restart message on the management channel.

Reply:
Awesome!